src/js/components/tablero/Filter.jsx

```diff
@@ -39,6 +53,8 @@ class Filter extends Component {
     if (categoryData) this.getCategoryRows(categoryData.endpoint);
     this.setState({ titlePopup: nameCategory || 'Add Filter' });
     this.setState({ categorySelected: nameCategory || '' });
+    // Value in the searchBar removed
+    document.getElementById('searchBar').value = '';
```

Comment:
Add a searchTerm to the state and set it here to empty value.

Reply:
done